submissions/franchukv/js-oop-frogger/app.js

```diff
@@ -0,0 +1,117 @@
+const CANVAS_HEIGHT = 606;
+const CANVAS_WIDTH = 505;
+const TILE_HEIGHT = 83;
+const TILE_WIDTH = 101;
+const ENEMY_SPEED = 80;
+const PLAYER_STAT = {
```

Comment:
What does "STAT" mean?
Do not use abbreviations that are not conventional in software development.

Reply:
It means "characteristics", just a wrong naming, fixed